docs/sources/tempo/introduction/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+aliases:
+  - /docs/tempo/getting-started/traces
+  - /docs/tempo/traces
+description: "What are traces? Learn about traces and how you can use them."
+keywords:
+  - Grafana
+  - traces
+  - tracing
+title: Introduction
+weight: 120
+---
+
+# Introduction
+
+A trace represents the whole journey of a request or an action as it moves through all the nodes of a distributed system, especially containerized applications or microservices architectures.
+This makes them the ideal observability signal for discovering bottlenecks and interconnection issues.
+
+Traces are composed of one or more spans.
+A span is a unit of work within a trace that has a start time relative to the beginning of the trace, a duration and an operation name for the unit of work.
+It usually has a reference to a parent span (unless it's the first span, the root span, in a trace).
+It frequently includes key/value attributes that are relevant to the span itself, for example the HTTP method used by the span, as well as other metadata such as the service name, sub-span events, or links to other spans.
```

Comment:
```suggestion
It frequently includes key/value attributes that are relevant to the span itself, for example the HTTP method used in the request, as well as other metadata such as the service name, sub-span events, or links to other spans.
```

Probably a nit: a span does not use a HTTP method, it sounds a bit strange to me.